src/prerender.ts

```diff
@@ -49,13 +49,16 @@ export async function prerender(nitro: Nitro) {
   // Build with prerender preset
   nitro.logger.info("Initializing prerenderer");
   nitro._prerenderedRoutes = [];
+  const prerendererConfig = { ...nitro.options._config };
```

Comment:
I think would be nice to pass the merged object (with static, rootDir, preset, logLevel) to this object to allow full control within hook